### ISSUE TITLE
make IdleConnection limit configurable

### ISSUE THREAD

**Issue by aberoham** [gravitational/teleport#1851]
**What happened**:

By default, idle Teleport sessions take ~20 minutes to time out.

**What you expected to happen**:

To be able to configure Teleport's idle connection timeout to be longer or shorter based on admin requirements. Notably, PCI DSS standards require that idle SSH sessions time out within fifteen minutes, see https://cisofy.com/compliance/pci-dss/pci-dss-8-1-8-session-idle-timeout/

Note: This is a Teleport commercial customer request, for their immediate use case simply being able to set the value cluster-wide would be sufficient. (They do not currently need the value to be configurable per-role)

**How to reproduce it (as minimally and precisely as possible)**:

n/a

**Environment**:
- Teleport version (use `teleport version`): 2.5.x
- Tsh version (use `tsh version`): n/a
- OS (e.g. from /etc/os-release): n/a

**Comment by klizhentas** [comment]
yeah, I would rather set it globally to 15 minutes now and call it a day
